boskos/cmd/mason/Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM alpine
+FROM google/cloud-sdk:alpine
 
-RUN apk update && apk upgrade && apk add ca-certificates && rm -rf /var/cache/apk/*
+RUN apk update \
+  && apk upgrade \
+  && apk add ca-certificates && rm -rf /var/cache/apk/* \
+  && gcloud components install kubectl -q --no-user-output-enabled
+
+RUN gcloud components install kubectl -q
```

Comment:
Is the repeatition intentional? The same command is included in the previous RUN clause

Reply:
you are right thanks!